DataPackages/Goal/1/datapackage.json

```diff
@@ -1,23 +1,26 @@
  {
-                    "name": "Goal 1",
+                    "name": "goal-1",
```

Comment:
https://goodtables.io/github/roll/SDG/jobs/1

Here goodtables.io has helped with a metadata rule - a data package name should be lower-cased.